default.json

```diff
@@ -126,6 +126,19 @@
       "recreateClosed": true,
       "schedule": "after 3:00 am and before 6:00 am every 2 weeks on Wednesday"
     },
+    {
+      "matchManagers": ["buildkite"],
+      "matchPackagePatterns": ["^seek-jobs/", "^seek-oss/"],
+
+      "stabilityDays": 0,
+      "updateNotScheduled": true
+    },
+    {
+      "matchManagers": ["buildkite"],
+      "matchPackageNames": ["seek-jobs/gantry"],
+
+      "schedule": "before 6:00 pm every weekday"
```

Comment:
I've temporarily set the Gantry rule to 6pm instead of 6am to try to get v1.6.0 PRs raised today. We can revert after this is sorted.